includes/CreateWiki/SpecialCreateWiki.php

```diff
@@ -26,6 +26,7 @@ public function __construct( CreateWikiHookRunner $hookRunner ) {
 	protected function getFormFields() {
 		$par = $this->par;
 		$request = $this->getRequest();
+		$this->checkReadOnly();
```

Comment:
This is also not needed, it is done by the default execute method of FormSpecialPage that is not overridden here.